engine/src/main/resources/swagger/cromwell.yaml

```diff
@@ -203,7 +204,6 @@ paths:
           in: path
           default: v1
         - name: start
-          description: Start Date
```

Comment:
No. No they weren't show up. They were causing cryptic errors on http://editor.swagger.io.

A few lines below there was a second key "description" with a different value. Due to massive debates in the internal yaml community regarding duplicate keys, parsers are wildly inconsistent. See comments in SwaggerServiceSpec for more info.

In regular snake yaml parsing, the latter key was overwriting this former. But when the valid-according-to-snake-yaml yaml was put into http://editor.swagger.io, the swagger site effectively just printed out "NOPE!" errors while I struggled to cross reference the swagger docs figuring out what the heck was wrong with a basic "description" tag.

I'm not bitter. ;)